app/views/posts/show.html.erb

```diff
@@ -13,6 +13,11 @@
     <div>
       <p class="mt-5 ml-3" ><%= @post.text %></p>
     </div>
+
+    <div class="mb-5 mt-5 ml-3 flex gap-5">
+      <a class="border-solid border-4 border-black cursor-pointer"><%= render partial: 'shared/add_like' %></a>
+      <a href="/users/<%= @user.id %>/posts/<%= @post.id %>/comments/new" class="border-4 border-solid border-black"> New comment</a>
+    </div>
   </div>
 
   <%= render partial: 'shared/comment', locals: {post: @post} %>
```

Comment:
- [ ] An error is thrown on the _Post#show_  page just after saving a comment for a given post. Let's ensure to fix it.

![image](https://github.com/rmiyoyo/blog-app/assets/12745474/d373e7c8-c8e3-4fb0-b40f-37b05e4bbc39)

Reply:
- [ ] As requested by the previous reviewer, kindly fix the error on the Post#show page. 

Videocast
[cinnamon-2023-12-23T230832+0200.webm](https://github.com/rmiyoyo/blog-app/assets/81932478/cc67fcd9-7f52-461b-ba59-8764bdfa8009)